### PR TITLE
CLOUDP-113304: Create "Post packaging smoke tests (Docker)" tasks for atlascli

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-113304

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](https://github.com/mongodb/mongocli/blob/master/e2e/E2E-TESTS.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

The goal of this PR is to add atlascli tasks in the `Post packaging smoke tests (Docker)` variant. This ticket is the 3nd of 4 tickets that aims to release atlascli (see [CLOUDP-113173](https://jira.mongodb.org/browse/CLOUDP-113173)).

### CODE REVIEW

File: .githooks/pre-commit [hunk 1]
```diff
@@ -19,7 +19,7 @@ if [[ -n "${STAGED_GO_FILES}" ]]; then
     git add docs
 fi
 
-STAGED_EVG_FILES=$(git diff --cached --name-only | grep "evergreen.yml$")
+STAGED_EVG_FILES=$(git diff --cached --name-only | grep -e "evergreen.yml$" -e "release.yml$")
```
Comment (andreaangiolillo) [inline .githooks/pre-commit:22]: drive-by fix

File: build/package/docker/atlascli-centos7-rpm.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM centos:7
+
+ARG revision
+ARG created_at
+ARG version
+
+ENV URL=https://mongodb-mongocli-build.s3.amazonaws.com/mongocli-master/dist/${revision}_${created_at}/mongodb-atlas-cli_${version}-next_linux_x86_64.rpm
```
Comment (threebee) [inline build/package/docker/atlascli-centos7-rpm.Dockerfile:7]: [q] do we need these Dockerfiles? Can't we make `URL` a `ARG` and pass it before scheduling the docker image run?

**Review by blva — APPROVED** [review]

File: build/package/docker/atlascli-centos7-rpm.Dockerfile (hunk 2 above)
Comment (threebee) [reply build/package/docker/atlascli-centos7-rpm.Dockerfile:7]: or just a arg which would decide if Dockerfile would run in `atlascli` or `mongocli` mode?

Comment (andreaangiolillo) [reply build/package/docker/atlascli-centos7-rpm.Dockerfile]: updated

**Review by threebee — APPROVED** [review]
LGTM! Thanks for the changes!
